### PR TITLE
Mobile 877

### PR DESCRIPTION
**Pull request by khmMouna**
### What do these changes do?
Break up the sample app into 2 screens: home and settings. Tie them together with bottom navigation.

### Why are these changes necessary?
To make the react-native sample app more organized.

#### Verification Screenshots:
**iOS**
home-iOS
settings-iOS

**Android**
home-android
settings-android

### CODE REVIEW

File: example/App.js [hunk 1]
```diff
@@ -123,12 +128,11 @@ const styles = StyleSheet.create({
 const notificationsEnabledKey = "com.urbanairship.notificationsEnabled"
 const locationEnabledKey = "com.urbanairship.locationEnabled"
 
-export default class AirshipSample extends Component {
+class SettingsScreen extends Component {
```
Comment (rlepinski) [inline example/App.js]: Can we break out the SettingsScreen and HomeScreen into their own files?

File: example/screens/HomeScreen.js [hunk 2]
```diff
@@ -0,0 +1,76 @@
+/**
+ * Sample React Native App
```
Comment (BrianBatchelder) [inline example/screens/HomeScreen.js]: Could you add a description of `HomeScreen`?

File: example/screens/SettingsScreen.js [hunk 3]
```diff
@@ -0,0 +1,348 @@
+/**
+ * Sample React Native App
```
Comment (BrianBatchelder) [inline example/screens/SettingsScreen.js]: Could you add a description of `SettingsScreen`?

File: scripts/run_ci_tasks.sh [hunk 4]
```diff
@@ -74,7 +74,7 @@ if $ANDROID ; then
     if [[ ! -f ${PROJECT_PLATFORM_PATH}/app/src/main/assets/airshipconfig.properties ]]; then
       cp -np ${PROJECT_PLATFORM_PATH}/app/src/main/assets/airshipconfig.properties.sample ${PROJECT_PLATFORM_PATH}/app/src/main/assets/airshipconfig.properties || true
     fi
-
+
```
Comment (BrianBatchelder) [inline scripts/run_ci_tasks.sh]: PRs are generally easier to review if they contain only changes relevant to the PR. When I create a PR, I first take a quick pass through it and eliminate unrelated changes such as this. It takes some effort as more and more editors do automatic re-formatting.
